src/System.Private.Xml.Linq/tests/misc/LoadSaveAsyncTests.cs

```diff
@@ -16,6 +16,7 @@ namespace CoreXml.Test.XLinq
 {
     public class LoadSaveAsyncTests : BridgeHelpers
     {
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]   //Async methods defined only on Core
```

Comment:
I'd have thought the correct way was what we do elsewhere: put tests using APIs specific to a particular netcore version in a file specific to that version, and only compile it in when targeting that version.